packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md

```diff
@@ -0,0 +1,180 @@
+
+* **What path should the webhooks be forwarded to (i.e.: /webhooks)?**
+
+    Enter: `/okta-webhooks-endpoint`
+
+* **What's the connection label (i.e.: My API)?**
+
+    Enter: `My Okta Response Server`
+
+With this data, Hookdeck creates a session and URL to use for sending requests. If you see the following content in your terminal, Hookdeck is running successfully:
+
+![A screen shot of a terminal that displays a Hookdeck session status. The session status contains urls that tunnel into the local port.](/img/hookdeck-and-event-hooks-session.png)
+
+> **Note:** Copy the forwarding **Webhook URL** that is available from the Hookdeck terminal session. For example, `https://events.hookdeck.com/e/src_s8lCGfojGBPj8L0lszAZl6fD`. You'll use this URL when setting up your Okta Event Hook.
+
+See [Hookdeck](https://hookdeck.com/) or their [Product docs](https://hookdeck.com/docs/introduction) for further information.
+
+### Review Hookdeck dashboard
+
+The Hookdeck dashboard provides an opportunity to review all calls to your local application. See [Browsing Events](https://hookdeck.com/docs/browsing-event) for details on using this interface. With the Hookdeck utility running, use the dashboard URL to access details on your hook calls: `https://dashboard.hookdeck.com/cli/events`.
+
+Each call to your local application appears in the dashboard and includes the response body, header, and other details:
+
+![A screen shot of the Hookdeck web dashboard that includes the response body, header, and other details.](/img/hookdeck-dashboard.png)
+
+## Create an Okta Event Hook
+
+Create the Okta Event Hook to work with your local application, which can now be exposed externally. The Event Hook must be set up and verified within your Okta Admin Console.
+
+### Set up the Event Hook
+
+1. Sign in to your [Okta org](https://login.okta.com/).
+
+2. From the Admin Console, go to **Workflow** > **Event Hooks**.
+
+3. Click **Create Event Hook**. The **Add Event Hook Endpoint** dialog box opens.
+
+4. In the **Name** field, add a unique name for the Hook (in this example, "New User Event Hook").
+
+5. In the **URL** field, add your external service URL from Hookdeck. For this example, use the **Webhook URL** highlighted after running Hookdeck, see [Run Hookdeck](#run-hookdeck). For example, your URL should appear similar to: `https://events.hookdeck.com/e/src_s8lCGfojGBPj8L0lszAZl6fD`.
+
+    >**Note:** Hookdeck generates the URL when creating the session and incorporates the local application's hook endpoint as part of the unique URL. That is, the endpoint is not explicitly defined as part of the URL.
+
+6. Leave the **Authentication field** and **Authentication secret** values blank in this example. However, to add Basic Authentication, add the application code required at [Add Basic Authorization and Body Parsing](/docs/guides/common-hook-set-up-steps/nodejs/main/#add-basic-authorization-and-body-parsing) and then enter the following values for those fields:
```

Comment:
Maybe add the a new line before the "However"? This is an alternative step that could confuse the user?